src/discordtipbot/discordbot.cr

```diff
@@ -133,6 +121,38 @@ class DiscordBot
       end
     end
 
+    # Handle new guilds and owner notifying etc
+    @streaming = false
+
+    @bot.on_ready do |payload|
+      # Only fire on first READY, or if ID cache was cleared
+      next unless @unavailable_guilds.empty?
```

Comment:
This should probably also check `@available_guilds.empty?` as both need to be properly reset to re-do streaming